bbb-common-web/src/main/java/org/bigbluebutton/presentation/imp/Office2PdfPageConverter.java

```diff
@@ -58,14 +51,43 @@ public static boolean convert(File presentationFile, File output, int page, Uplo
       String logStr = gson.toJson(logData);
       log.info(" --analytics-- data={}", logStr);
 
-      final DocumentFormat sourceFormat = DefaultDocumentFormatRegistry.getFormatByExtension(
-              FilenameUtils.getExtension(presentationFile.getName()));
+//      This method using Jod as office converter was replaced by a customizable script (solving issue https://github.com/bigbluebutton/bigbluebutton/issues/10699)
+//      final DocumentFormat sourceFormat = DefaultDocumentFormatRegistry.getFormatByExtension(
+//              FilenameUtils.getExtension(presentationFile.getName()));
+//
+//      inputStream = new FileInputStream(presentationFile);
+//      outputStream = new FileOutputStream(output);
+//
+//      converter.convert(inputStream).as(sourceFormat).to(outputStream).as(DefaultDocumentFormatRegistry.PDF).execute();
+//      outputStream.flush();
 
-      inputStream = new FileInputStream(presentationFile);
-      outputStream = new FileOutputStream(output);
+      try {
+        log.info("Calling conversion script " + presOfficeConversionExec);
 
-      converter.convert(inputStream).as(sourceFormat).to(outputStream).as(DefaultDocumentFormatRegistry.PDF).execute();
-      outputStream.flush();
+        if(presOfficeConversionExec == null) throw new Exception("Cannot find the conversion script path.");
+
+        File conversionScript = new File(presOfficeConversionExec);
+        if(!conversionScript.exists()) throw new Exception(presOfficeConversionExec + ", file not found.");
+
+        Process p = Runtime.getRuntime().exec(String.format(presOfficeConversionExec + " %s %s", presentationFile.getAbsolutePath(), output.getAbsolutePath()));
+
+        BufferedReader stdInput = new BufferedReader(new InputStreamReader(p.getInputStream()));
+        BufferedReader stdError = new BufferedReader(new InputStreamReader(p.getErrorStream()));
+        String s = null;
+
+        // read the output from the command
+        while ((s = stdInput.readLine()) != null) {
+          log.info(s);
+        }
+
+        // read any errors from the attempted command
+        while ((s = stdError.readLine()) != null) {
+          log.error(s);
+        }
+
+      } catch (IOException e) {
+        log.error("Exception while calling convert script: " + e.getMessage(), presentationFile.getName());

```

Comment:
Add a prefix to this log ( so we know from which conversion it was dispatched ).